content/docs/web-components.md

```diff
@@ -6,11 +6,11 @@ redirect_from:
   - "docs/webcomponents.html"
 ---
 
-React and [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) are built to solve different problems.  Web Components provide strong encapsulation for reusable components, while React provides a declarative library that keeps the DOM in sync with your data. The two goals are complementary. As a developer, you are free to use React in your Web Components, or to use Web Components in React, or both.
+React y [Web Components](https://developer.mozilla.org/es/docs/Web/Web_Components) estan construidos para resolver diferentes problemas.  `Web Components` proporciona una fuerte encapsulación para componentes reutilizables, mientras que React proporciona una biblioteca declarativa que mantiene el DOM sincronizado con sus datos. Los dos objetivos se complementan. Como desarrollador, eres libre de usar React en tus `Web Components`, utilizar `Web Components` en React, o ambos.
```

Comment:
Thank  you @Paolore. here some suggestions :=)
* Let's keep the link to https://developer.mozilla.org/en-US/docs/Web/Web_Components documentation in English for now, since the Spanish one is not ready yet.
* `están construidos...` it has accent.
* `DOM sincronizado con tus datos.` we agreed on using `tú`.

Reply:
We will late have to check with the other members, but for now, we decided always to use the MDN Spanish links.